x/wasm/types/types_test.go

```diff
@@ -178,7 +178,7 @@ func TestContractInfoMarshalUnmarshal(t *testing.T) {
 
 	anyTime := time.Now().UTC()
 	// using gov proposal here as a random protobuf types as it contains an Any type inside for nested unpacking
-	myExtension, err := govtypes.NewProposal(&govtypes.TextProposal{Title: "bar"}, 1, anyTime, anyTime)
+	myExtension, err := govtypes.NewProposal(&govtypes.TextProposal{Title: "bar"}, 1, anyTime, anyTime, false)
```

Comment:
I thought that we decided to instead revert the breaking interfaces and introduce a decorator pattern. This is to avoid breaking dependencies such as wasmd.

For example, we would have an old constructor but a function `govtypes.NewProposal(...).WithExpedited(true)`

Reply:
Sorry, I wasn't tracking this decision/discussion. I basically just added the minimum to the proposal lines in order to utilize the newest sdk fork. Can you go more into what needs to be done to achieve what you are suggesting?